### PR TITLE
Update deps to latest.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ for(const module of contextModules) {
   for(const c in contextConstants) {
     const contextUrl = contextConstants[c];
     bedrockConstants[c] = contextUrl;
-    jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
+    if(c.includes('_URL')) {
+      jsonLdDocumentLoader.addStatic(contextUrl, contexts.get(contextUrl));
+    }
   }
```
Comment (davidlehn) [inline lib/index.js]: This looks fragile and awkward.  The contexts are in their own map.  How about rewriting that whole loop as:
```js
  for(const c in contextConstants) {
    bedrockConstants[c] = contextConstants[c];
  }
  for(const [url, context] of contexts) {
    jsonLdDocumentLoader.addStatic(url, context);
  }
```

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: lib/index.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/index.js]: :+1: seems like a decent idea.

Comment (mattcollier) [reply lib/index.js]: TY. Implemented.

**Comment by mattcollier** [comment]
Waiting on https://github.com/veres-one/veres-one-context/pull/12 or a PR on the 12.x branch which does not currently exist.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**